src/dotnet/CommonLocalizableStrings.cs

```diff
@@ -179,5 +179,9 @@ internal class CommonLocalizableStrings
         public const string SpecifiedNameExists = "Specified name {0} already exists. Please specify a different name.";
         public const string SpecifiedAliasExists = "Specified alias {0} already exists. Please specify a different alias.";
         public const string MandatoryParameterMissing = "Mandatory parameter {0} missing for template {1}. ";
+
+        public const string ProjectNotCompatibleWithFrameworks = "Project `{0}` cannot be added due to incompatible targeted frameworks between the two projects. Please review the project you are trying to add and verify that is compatible with the following targets:";
```

Comment:
I don't believe so for two reasons:
- rest of the string is being generated
- rest of the string is not supposed to be localized